utils/src/requests/user/update.dto.ts

```diff
@@ -0,0 +1,53 @@
+import { IsEmail, IsString, IsOptional, Matches } from 'class-validator';
+import { ENG_REGEX, NUM_REGEX, validationMessages, validationOptionsMsg } from '../../validation.util';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UpdateDTO {
+  @ApiProperty({
```

Comment:
This field is optional
```suggestion
  @ApiPropertyOptional({
```